frame/nomination-pools/src/lib.rs

```diff
@@ -1276,6 +1517,10 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type MaxPoolMembersPerPool<T: Config> = StorageValue<_, u32, OptionQuery>;
 
+	/// Maximum commission that can be set for all pools.
```

Comment:
I think a `set_global_max_commission` call is missing?

Reply:
it is part of `set_config`.